tests/conftest.py

```diff
@@ -77,6 +77,10 @@ def mock_login_token_response():
         TimerScopes.resource_server: _mock_token_response_data(
             TimerScopes.resource_server, TimerScopes.timer
         ),
+        "flows.globus.org": _mock_token_response_data(
+            "flows.globus.org",
+            "https://auth.globus.org/scopes/eec9b274-0c81-4334-bdc2-54e90e689b9a/view_flows",  # noqa E501
+        ),
```

Comment:
You may have noticed that the data above this is not uniform. The `TimerScopes` usage is the newest, and I think I'd like for us to follow that pattern (and go back to fix the other data here). If you agree that it's better, we'd get
`FlowsScopes.resource_server: _mock...(FlowsScopes.resource_server, FlowsScopes.view_flows)`.

Also note that this should match whatever we're requesting in `LoginManager`, if that should change.

Reply:
I think I like the explicit string definitions here.

Generally speaking I'd prefer referencing resource server like is done in the login manager. But because this is a test not source, explicitly putting the resource server/expected scope strings here relies less on upstream definitions being correct (ie. the flows resource server) and actually catches more issues that could occur (like the fix we needed in sdk).

On the flip side, supposing these values changed in real life, the impact would just be a test suite breaks & is caught on automated test run before the cli goes out to customers. I'd argue the value of confirming our expectation of how things work is worth the cost of a very occasional dev process slowdown.

Reply:
Yep, I'm convinced. 👍  I'll mark this resolved.